src/feedback/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import styled from "styled-components";
+import Alert, {
+  alertKindValues,
+  alertSeverityValues,
+} from "src/feedback/Alert/Alert";
+import { heading1 } from "src/guidelines/theme";
+import { isVowel, ucfirst } from "src/utils/stringHelper";
+
+const meta: Meta<typeof Alert> = {
+  component: Alert,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Alert>;
+
+const Title = styled.h1`
+  ${heading1};
+`;
+
+const FlexContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const AlertDemo: Story = {
+  args: {
+    severity: "info",
+    kind: "normal",
+    closeable: false,
+    children: "Message",
+  },
+  argTypes: {
+    severity: {
+      options: alertSeverityValues,
+      mapping: { alertSeverityValues },
+      control: { type: "select" },
+      description: "",
+    },
+    kind: {
+      options: alertKindValues,
+      mapping: { alertKindValues },
+      control: { type: "select" },
+      description: "",
+    },
+  },
+};
+
+export const AlertList = () => {
+  return (
```

Comment:
If we want to be consistent, you should add a Title here ;-)
`import { Title } from "src/utils/StorybookComponents/Titles"; 
`
`<Title>Toast</Title>`